tests/unit/db/result_tests.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+result module - Unit tests for Result class
+"""
+import unittest
+
+from cloudant.design_document import DesignDocument
+from cloudant.errors import CloudantArgumentError
+from cloudant.result import python_to_couch, Result
+from cloudant.views import View
+from tests.unit.db.unit_t_db_base import UnitTestDbBase
+
+
+class PythonToCouchTests(unittest.TestCase):
+    """
+    Test cases for validating python_to_couch's options
+    """
+
+    # TODO more python_to_couch test cases to be added to completely cover issue #42
+
+    def test_valid_option_group_level(self):
+        """
+        Test group_level option is valid
+        """
+        opts = {'group_level': 100}
+        translation = python_to_couch(opts)
+        self.assertEqual(translation['group_level'], 100)
+
+    def test_invalid_option_group_level(self):
+        """
+        Test group_level options are invalid
+        """
+        opts = {'group_level': True}
+        self.assertRaises(CloudantArgumentError, python_to_couch, opts)
+        opts = {'group_level': '100'}
+        self.assertRaises(CloudantArgumentError, python_to_couch, opts)
+
+class ResultTests(UnitTestDbBase):
+    """
+    Result unit tests
+    """
+
+    # TODO more Result test cases to be added to completely cover issue #42
+
+    def setUp(self):
+        """
+        Set up test attributes
+        """
+        super(ResultTests, self).setUp()
+        self.db_set_up()
+        self.ddoc = DesignDocument(self.db, 'ddoc001')
+        self.ddoc.add_view(
+            'view001',
+            'function (doc) {\n  emit(doc._id, 1);\n}',
+            '_count'
+        )
+        self.ddoc.save()
+
+    def tearDown(self):
+        """
+        Reset test attributes
+        """
+        self.db_tear_down()
+        super(ResultTests, self).tearDown()
+
+    def test_constructor(self):
+        """
+        Test instantiating a Result
+        """
+        result = Result(self.ddoc.get_view("view001"),
+                        startkey='1', endkey='9', page_size=1000)
```

Comment:
The typical rule for indentation in Python is 4 spaces.  I suggest changing the statement to be either:

```python
result = Result(self.ddoc.get_view('view001'),
    startkey='1', endkey='9', page_size=1000)
```

or 

```python
result = Result(
    self.ddoc.get_view('view001'),
    startkey='1',
    endkey='9',
    page_size=1000
)
```